tests/ExportTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use RonasIT\Support\Traits\FixturesTrait;
+
+class ExportTest extends \Orchestra\Testbench\TestCase
```

Comment:
please use class names aliases instead

```suggestion
class ExportTest extends TestCase
```